akka-stream-tests/src/test/scala/akka/stream/impl/SynchronousIterablePublisherSpec.scala

```diff
@@ -1,172 +0,0 @@
-/**
- * Copyright (C) 2014 Typesafe Inc. <http://www.typesafe.com>
- */
-package akka.stream.impl
-
-import scala.collection.immutable
-import scala.concurrent.duration._
-import akka.stream.testkit.AkkaSpec
-import akka.stream.testkit.StreamTestKit
-import akka.testkit.TestProbe
-import org.reactivestreams.{ Subscriber, Subscription }
-
-class SynchronousIterablePublisherSpec extends AkkaSpec {
-
-  "A SynchronousPublisherFromIterable" must {
```

Comment:
I moved everything from this file into FlowIteratorSpec.scala, or did I miss something?